NitroxServer/Serialization/World/WorldPersistence.cs

```diff
@@ -197,18 +199,31 @@ private World CreateFreshWorld()
                                   List<NitroxInt3> parsedBatchCells,
                                   List<EscapePodModel> escapePods,
                                   StoryTimingData storyTimingData,
-                                  ServerGameMode gameMode)
+                                  ServerGameMode gameMode,
+                                  string seed)
         {
+            if (string.IsNullOrWhiteSpace(seed))
+            {
+                seed = config.Seed;
```

Comment:
For development, we should always use the same seed.

Reply:
shall i add a
```csharp
#if DEBUG

#elseif

#endif
```?

Reply:
yeah you can add an `#if debug` to manually se the seed.